package.json

```diff
@@ -8,23 +8,23 @@
     "repository": "git+https://github.com/Doist/reactist.git",
     "prettier": "@doist/prettier-config",
     "main": "dist/index.js",
-    "types": "dist/index.d.ts",
+    "module": "es/index.js",
+    "typings": "lib/index.d.ts",
     "files": [
         "dist",
-        "src"
+        "es",
+        "styles"
     ],
     "engines": {
         "node": ">=10"
     },
     "scripts": {
         "start": "tsdx watch",
-        "build": "npm run build-tsdx && npm run build-webpack; rimraf 'dist/umd'",
-        "build-webpack": "NODE_ENV=production ./node_modules/.bin/webpack --progress --color",
-        "build-tsdx": "tsdx build --tsconfig tsconfig.dist.json",
-        "build-watch": "tsdx watch",
-        "build-types": "rimraf 'lib/**/*.d.ts'; npx tsc -p tsconfig.lib.json;",
-        "build-storybook": "build-storybook -o docs",
-        "build-all": "npm run build && npm run build-storybook",
+        "build-all": "npm run build && npm run build:storybook",
+        "build": "npm run clean; tsdx build --tsconfig tsconfig.dist.json; npm run build:reorganize-styles",
+        "build:reorganize-styles": "rimraf dist/assets; mv es/index.css es/reactist.css; mkdir styles; find es -iname '*.css' -type f -exec mv {} styles/ \\;",
```

Comment:
Do you think it would make sense to put this into a bash script (separate file) and add a couple of comments?